Formula/trojan-go.rb

```diff
@@ -0,0 +1,94 @@
+class TrojanGo < Formula
+  desc "Trojan proxy in Go"
+  homepage "https://p4gefau1t.github.io/trojan-go/"
+  url "https://github.com/p4gefau1t/trojan-go.git",
+      tag:      "v0.10.0",
+      revision: "a9ce6d4ada9143d58e8d1373d23f9ebfe871119e"
+  license "GPL-3.0-only"
+  head "https://github.com/p4gefau1t/trojan-go.git"
+
+  depends_on "go" => :build
+
+  resource "geoip" do
+    url "https://github.com/v2fly/geoip/releases/download/202105070029/geoip.dat"
+    sha256 "f3aeb80c21fd403c6030988477d0cac7574322b786733d0ecbc24ff4ab55ce93"
+  end
+
+  resource "geosite" do
+    url "https://github.com/v2fly/domain-list-community/releases/download/20210508061759/dlc.dat"
+    sha256 "3ad770409f62c9566dd07661c1f9cda47dfd6469f39019c90530b9acbf78dc25"
+  end
+
+  def install
+    commit = Utils.git_head
+    execpath = libexec/name
+    ldflags = %W[
+      -X github.com/p4gefau1t/trojan-go/constant.Version=v#{version}
+      -X github.com/p4gefau1t/trojan-go/constant.Commit=#{commit}
+    ]
```

Comment:
```suggestion
    execpath = libexec/name
    ldflags = %W[
      -X github.com/p4gefau1t/trojan-go/constant.Version=v#{version}
      -X github.com/p4gefau1t/trojan-go/constant.Commit=#{Utils.git_head}
    ]
```
You only use `commit` in one place, so there's no need to define an extra variable for it.

Reply:
Resolved.